stats/index.js

```diff
@@ -12,9 +12,8 @@ for (var j = 0; j < 100; ++j) {
   let stages = []
 
   for (var i = 1; i < 4; ++i) {
-    let utxos = Simulation.generateTxos(20 / i, min, max)
-    let txos = Simulation.generateTxos(80 / i, min, max / 3)
-
+    let utxos = Simulation.generateTxos(20 / i, min, max, {106: 0.9, 300: 0.1})
```

Comment:
Hm, I really don't know where it came from. (It was originally in simulation.js.) It seems like an arbitrary number to simulate larger P2SH inputs.... that can really have arbitrary lenghts.

I might look at blockchain how long are p2sh inputs on average in the last ~2 years?